galgebra/printer.py

```diff
@@ -1089,9 +1091,16 @@ def LatexFormat(Fmode=True, Dmode=True, ipy=False):
 def Get_Program(off=False):
     global off_mode
     off_mode = off
+    # galgebra 0.5.0
+    warnings.warn(
+        "galgebra.printer.Get_Program is deprecated, and exists solely to "
+        "toggle whether galgebra.printer.Print_Function does anything. If you "
+        "want to turn off program printing, then just don't call Print_Function!",
+        DeprecationWarning, stacklevel=2)
 
 
 def Print_Function():
+    """ Print out the source of the current function """
     if off_mode:
         return
```

Comment:
Why not remove this as well?

Reply:
Because that would be a breaking change - once we remove `Get_Program`, we can remove this.

Reply:
I could stick a deprecation warning here too, but that would probably be overkill

Reply:
Oh

Reply:
Let's go ahead and merge as-is. Sorry I wasn't thinking clearly. Now it doesn't seem to be a problem.